templates/site/remote-processing.html

```diff
@@ -44,24 +44,15 @@
         {% set doc_link = "https://pavics-sdi.readthedocs.io/projects/raven/en/latest/" %}
         {% set content %}
             <p>
-                Raven offers processes related to hydrological modeling, and in particular, the Raven
-                hydrological modeling framework.
+                Raven is an open source server project offering physiographic analysis capabilities over North 
```

Comment:
@huard Raven only works for North America?

Reply:
The datasets that are connected to it only cover NA.

Reply:
But is that because those are the datasets loaded on PAVICS or are those datasets built in to the raven wps code?

Reply:
They're loaded on PAVICS' Geoserver, and used as defaults for processes. So if you have a public link to an alternate database, Raven should still work, although we haven't really tested processes with datasets other than the defaults.

Reply:
Ok that makes sense. For this page, let's remove the reference to North America then. 
This page is meant to describe raven in general, not the specific raven + geoserver configuration on a specific node (PAVICS)